conf/messages

```diff
@@ -24,7 +24,6 @@ common.use = use
 common.error.errorKey = Error:
 common.opensInNewTab = (opens in new tab)
 common.currency.hint = For example, £193.52
-common.currency.hint1 = For example, £193.54
```

Comment:
is this not used by any other page? I remember seeing this hint in someone pages too

Reply:
I confirmed that `hint1` is only used in that page. It's `hint` which is still used.

Reply:
The fact that sbt it:test passed, is enough to prove this.